subjects/PE/PPC/hello_ppc.reko/hello_ppc_text.c

```diff
@@ -1250,11 +1250,11 @@ Eq_n fn004020BC(Eq_n r2, struct Eq_n * r3, ptr32 r29, ptr32 r30, struct Eq_n * r
 				word32 ctr_n;
 				for (ctr_n = 0x05; ctr_n != 0x00; --ctr_n)
 				{
-					(&fp->aFFFFFFA0->dw0000)[r11_n] = (struct Eq_n) 0x00;
+					Mem69[fp + ~0x5F + r11_n:word32] = 0x00;
 					int32 r11_n = r11_n + 0x04;
-					(&fp->aFFFFFFA0->dw0000)[r11_n] = (struct Eq_n) 0x00;
-					(&fp->aFFFFFFA0->dw0000)[r11_n + 0x04] = (struct Eq_n) 0x00;
-					(&fp->aFFFFFFA0->dw0000)[r11_n + 0x08] = (struct Eq_n) 0x00;
+					Mem71[fp + ~0x5F + r11_n:word32] = 0x00;
```

Comment:
Don't see a regression here. These are all minor improvements. Clearly `Mem71[fp + ~0x5F...` should be a stack variable, but that transformation should have happened during data flow analysis, so no regression.